### ISSUE TITLE
Failed to fetch file gs://chromium-clang-format

### ISSUE THREAD

**Issue by aisouard** [aisouard/libwebrtc#38]
It's happening more frequently with Travis under macOS.
Perhaps it doesn't like fetching multiple binaries in parallel ?

```
0> Failed to fetch file gs://chromium-clang-format/c44ad67f6d04cdd2e321d980488afa3e71a8ec79 for src/buildtools/mac/clang-format, skipping. [Err: Traceback (most recent call last):
  File "/Users/travis/build/aisouard/libwebrtc/out/depot_tools/src/depot-tools/gsutil.py", line 160, in <module>
    sys.exit(main())
  File "/Users/travis/build/aisouard/libwebrtc/out/depot_tools/src/depot-tools/gsutil.py", line 157, in main
    clean=args.clean)
  File "/Users/travis/build/aisouard/libwebrtc/out/depot_tools/src/depot-tools/gsutil.py", line 125, in run_gsutil
    gsutil_bin = ensure_gsutil(force_version, target, clean)
  File "/Users/travis/build/aisouard/libwebrtc/out/depot_tools/src/depot-tools/gsutil.py", line 95, in ensure_gsutil
    os.makedirs(target)
  File "/usr/local/Cellar/python/2.7.12_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/os.py", line 157, in makedirs
    mkdir(name, mode)
OSError: [Errno 17] File exists: '/Users/travis/build/aisouard/libwebrtc/out/depot_tools/src/depot-tools/external_bin/gsutil'
]
```

**Comment by aisouard** [comment]
The scripts are executed exactly at the same time, the directory should be created before running `download_from_google_storage.py`

**Comment by aisouard** [comment]
Fixed in 53a331a
